Source/Data/05 - DefaultSettings.sql

```diff
@@ -856,4 +856,20 @@ INSERT INTO CellCarrier (Name,Transform) VALUES
 ('Cricket Wireless', '{0}@mms.cricketwireless.net'),
 ('Republic Wireless','{0}@text.republicwireless.com'),
 ('Google Fi','{0}@msg.fi.google.com')
+GO
+
+INSERT INTO [MiMD.DBCleanUpTask] (SQLCommand, Schedule, Name) VALUES
+(
+'WITH TempTable as (
+	SELECT Max(TimeOfFailure) as MaxTimeOfFailure, FileGroupID
+	FROM DataOperationFailure
+	Group By (FileGroupID)
+)
+
+DELETE FROM DataOperationFailure WHERE ID in (
+	SELECT DataOperationFailure.ID FROM DataOperationFailure
+	JOIN TempTable ON TempTable.FileGroupID = DataOperationFailure.FileGroupID
+	Where DATEADD(DAY, -1, MaxTimeOfFailure) > DataOperationFailure.TimeOfFailure
+	)
+GO', '0 0 1 * *', 'DataOperations Failures Cleanup')
```

Comment:
You don't need the `GO`, and it might actually break the query.

https://learn.microsoft.com/en-us/sql/t-sql/language-elements/sql-server-utilities-statements-go?view=sql-server-ver16
> SQL Server provides commands that are not Transact-SQL statements, but are recognized by the **sqlcmd** and **osql** utilities and SQL Server Management Studio Code Editor. These commands can be used to facilitate the readability and execution of batches and scripts.

Reply:
It doesn't break it, but we can remove it if you'd like.